### PR TITLE
Delete aliases when deleting account in account service

### PR DESCRIPTION
**Pull request by ChristophWurst**
Supersedes #959

### CODE REVIEW

**Comment by ChristophWurst** [comment]
cc @PanCakeConnaisseur for review :)

File: lib/Db/AliasMapper.php [hunk 1]
```diff
@@ -64,4 +66,24 @@ public function findAll(int $accountId, string $currentUserId): array {
...
+	 * @param int $accountId the account whose aliases will be deleted
+	 * @param string $currentUserId the user that is currently logged in
+	 */
+	public function deleteAll($accountId, $currentUserId) {
+		$qb = $this->db->getQueryBuilder();
+
+		$query = $qb->delete($this->getTableName(), 'aliases')
+			->join('aliases', 'mail_accounts', 'accounts', $qb->expr()->eq('aliases.account_id', 'accounts.id'))
+			->where($qb->expr()->eq('account_id', $qb->createNamedParameter($accountId)))
+			->where(
```
Comment (rullzer) [inline lib/Db/AliasMapper.php]: andWhere?

Comment (skjnldsv) [reply lib/Db/AliasMapper.php]: What a suspense!

Comment (skjnldsv) [reply lib/Db/AliasMapper.php]: Ah nooo, it continues after. I'm disappointed, I thought it was a cliffhanger! :grin:

**Pull request merged**